lib/nexmo_developer/app/presenters/page_title.rb

```diff
@@ -6,14 +6,47 @@ def initialize(product, document_title)
 
   def title
     if @product && @document_title
-      "#{@product.titleize} > #{@document_title} | #{default_title}"
+      "#{normalized_product_title(@product)} > #{@document_title} | #{default_title}"
     elsif @document_title
       "#{@document_title} | #{default_title}"
     else
       default_title
     end
   end
 
+  def normalized_product_title(product)
+    if product.include?('/')
```

Comment:
@mheap @benhayehudi  can we just add a new entry to the existing `config/products.yml` named `title` or something along those lines (we can even fall back to `titleize` if it is not present) instead?

This looks extremely complex and quite hard to follow

Reply:
I'll defer to both of you, and happy to rearrange if need be.

Reply:
I will add though that while the code above is a little bit involved (albeit not too much) it doesn't change any of the actual page title implementation details. If we add a `title` or the like to `products.yml`, we'll need to adjust how the page title is derived, adding additional logic to check for its presence and use it instead of the current formulation. In other words, both approaches (the one in this PR) and the one suggested will involve some complexity.

Reply:
I can see arguments for both approaches. Adding a `title` entry in `products.yml` gives us the most flexibility, but also requires the end user to configure things each time a new product is added.

If this logic is only being used for page titles, let's switch to Fabian's recommendation. If it's used more generically, let's keep the upper/lower config

Reply:
that's a very good point, @mheap. Can we settle on the likelihood that users of `station` will want to use specialized lowercase or uppercase words throughout their text? I think that's a strong possibility to develop for.